dozer/cogs/moderation.py

```diff
@@ -13,6 +13,7 @@
 from ._utils import *
 from .. import db
 
+MAX_PURGE = 10000
```

Comment:
I'd say thats a tad bit too high imo, maybe 1000 or less would be more reasonable as we just saw what 1000 looked like